arangod/V8Server/v8-vocbase.cpp

```diff
@@ -1775,11 +1775,6 @@ static void JS_DropDatabase(v8::FunctionCallbackInfo<v8::Value> const& args) {
     TRI_V8_THROW_EXCEPTION(TRI_ERROR_ARANGO_USE_SYSTEM_DATABASE);
   }
 
-  if (!ExecContext::current().isAdminUser() || (ServerState::readOnly() && !ExecContext::current().isSuperuser())) {
-    events::DropDatabase("", TRI_ERROR_FORBIDDEN);
-    TRI_V8_THROW_EXCEPTION(TRI_ERROR_FORBIDDEN);
-  }
-
```

Comment:
I guess this can be removed here because there is a (slightly different) check in arangodb::methods::Databases::drop(...)?